content/en/blog/2023/egress-sni/index.md

```diff
@@ -53,7 +53,35 @@ internal destination of the gateway-implemented TCP proxy to the internal SNI fo
 
 The end-to-end request flow is shown in the following diagram:
 
-{{< image width="90%" link="./egress-sni-flow.png" alt="Egress SNI routing with arbitrary domain names" title="Egress SNI routing with arbitrary domain names" caption="Egress SNI routing with arbitrary domain names" >}}
+{{< image width="90%" link="./egress-sni-flow.svg" alt="Egress SNI routing with arbitrary domain names" title="Egress SNI routing with arbitrary domain names" caption="Egress SNI routing with arbitrary domain names" >}}
+
+In this diagram, egress HTTPS/TLS routes with Istio for arbitrary domain names, using SNI as a routing key.
```

Comment:
This diagram shows an egress HTTPS request to `en.wikipedia.org` using SNI as a routing key.